csrc/include/natten/cuda/fna/kernel_forward.h

```diff
@@ -129,14 +128,14 @@ struct FusedNeighborhoodAttentionKernel {
   // MapTileSizeToNd<kKeysPerBlock, NADim>::value;
   static constexpr bool kHasCausalDims = CausalMask::AnyCausalDims;
   static constexpr bool kSupportsRPB = kSupportsRPB_;
-  static constexpr bool kStoresLSE = kStoresLSE_;
   static_assert(
       !kSupportsRPB || !kHasCausalDims,
       "Causal NA does not support RPB yet.");
 
   using scalar_t = scalar_t_;
   using accum_t = float;
   using lse_scalar_t = float;
+  using max_scalar_t = float;
```

Comment:
Let's not introduce different element types for LSE and max. This was a mistake in the kernel to begin with.

It's assuming `lse_scalar_t` is the same as `accum_t`. Just replace both `lse_scalar_t` and `max_scalar_t` with `accum_t`, and remove these.